components/sysutils/chezmoi/patches/01-only-build-illumos.patch

```diff
@@ -0,0 +1,24 @@
+--- chezmoi-2.7.3/Makefile.orig	2021-10-22 15:34:17.000000000 +0000
++++ chezmoi-2.7.3/Makefile	2021-10-29 09:59:04.758384704 +0000
+@@ -2,17 +2,15 @@
+ GOLANGCI_LINT_VERSION=$(shell grep GOLANGCI_LINT_VERSION: .github/workflows/main.yml | awk '{ print $$2 }')
+ 
+ .PHONY: default
+-default: run build test lint format
++default: build
+ 
+ .PHONY: install
+ install:
+-	go install -ldflags "-X main.version=$(shell git describe --abbrev=0 --tags) \
+-		-X main.commit=$(shell git rev-parse HEAD) \
+-		-X main.date=$(shell date -u +%Y-%m-%dT%H:%M:%SZ) \
+-		-X main.builtBy=source"
++	go install
```

Comment:
Is there a reason we skip backing in the metadata into the binary?

Reply:
Yes. For some reason, including the metadata results in an error that the binary is "not in dotted-tri format". Just using `go install` without metadata resolves this issue